packages/taquito-core/src/errors.ts

```diff
@@ -55,7 +55,7 @@ export class InvalidAddressError extends ParameterValidationError {
     super();
     this.name = 'InvalidAddressError';
     this.message = `Invalid address "${address}"`;
-    errorDetail ? (this.message += ` ${errorDetail}.`) : null;
+    this.message += errorDetail ? ` ${errorDetail}.` : '.';
```

Comment:
what does an example of an error message for this look like?

Reply:
something like this `Invalid address "tz1iedjFYksExq8snZK9MNo4AvXHG" failed checksum.`